### PR TITLE
Add more improvements for i18n support 

### PR DESCRIPTION
**Pull request by macagua**
- Add i18n support for ZPT file
- Moved this translation into cpskin.locales package

### CODE REVIEW

File: cpskin/menu/browser/menu.pt [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <ul tal:replace="structure python: view.superfish_portal_tabs()" />
 
 <a href="#" id="search-btn"></a>
-<div id="mobnav-btn">Menu de navigation</div>
+<div id="mobnav-btn" i18n:translate="">Navigation menu</div>
```
Comment (bsuttor) [inline cpskin/menu/browser/menu.pt:12]: I suppose I have to compile and add this in cpskin.locales (fr) after merging it

Comment (macagua) [reply cpskin/menu/browser/menu.pt:12]: @bsuttor Right now I'm doing those changes on cpskin.locales package and I'll mixing it into the French language

Comment (macagua) [reply cpskin/menu/browser/menu.pt:12]: @bsuttor Done with commit [9c180b7](https://github.com/IMIO/cpskin.locales/pull/3/commits/9c180b7601058ad4e446a5b984c38bdeae64d1ba)

**Comment by macagua** [comment]
@bsuttor please, can you review this pull request and If all ok merge it?

**Comment by macagua** [comment]
@bsuttor Thanks :-)

**Pull request merged**